Chell0/machtbewwust
# #ft-create-components

# Pull Request: Add New Components

## Description
This pull request includes the following new components:
- `About`
- `Banner`
- `BlogCard`
- `Category`
- `Datenbank`
- `Footer`
- `Kalender`
- `MaterialCards`
- `NavBar`
- `Newsletter`
- `Pagination`
- `Section`
- `ShowMoreButton`
- `Tags`

These components have been developed and tested locally.

## Checklist
- [ ] All components have been added
- [ ] New dependencies are added to the `package.json`

## How to Test
1. Run the project and navigate through the application to see the new components in action.
2. Ensure each component renders correctly and interacts as expected.
